doc/build/html/_sources/index.rst.txt

```diff
@@ -0,0 +1,96 @@
+.. _root:
+
+Welcome to ZCash v1.0.15 Documentation
```

Comment:
Out-of-date, and this will easily get out-of-sync with `README`. Two options:

- Update `zcutil/make-release.py` to know about this file in order to update the version.
- (Preferred) auto-generate this file from `README`.